### ISSUE TITLE
Images don't work with _ and probably other characters

### ISSUE THREAD

**Issue by onen9ne** [Requarks/wiki#855]
**Describe the bug**

Uploading images does not work with spaces and _. I did not test it with other characters. Spaces show nothing. _ show the broken image with the alt text. Also, as you can see by the image that is broken when using dark mode.

**To Reproduce**
Upload an image with a space or named_like_this and it will not work.

**Expected behavior**
It to work no matter what you name your image. If it has spaces in it, substitute it with something else.

**Screenshots**

![sss](https://user-images.githubusercontent.com/51209690/58653330-15668600-82db-11e9-90b8-030380a15fe9.PNG)

**Host Info (please complete the following information):**
beta.148

**Comment by NGPixel** [comment]
I was able to upload and see images with underscores. Do you get any error in the console or network tab of your browser dev tools?

**Comment by onen9ne** [comment]
> I was able to upload and see images with underscores. Do you get any error in the console or network tab of your browser dev tools?

```
Failed to load resource: the server responded with a status of 404 (Not Found)
dark_construct.png:1 GET https://wiki.domain.com/dark_construct.png 404 (Not Found)
Image (async)
ai @ app.4eb2e32d4cd072ffaa61.js:117
k @ app.4eb2e32d4cd072ffaa61.js:117
(anonymous) @ app.4eb2e32d4cd072ffaa61.js:117
k @ app.4eb2e32d4cd072ffaa61.js:117
(anonymous) @ app.4eb2e32d4cd072ffaa61.js:117
e._update @ app.4eb2e32d4cd072ffaa61.js:117
r @ app.4eb2e32d4cd072ffaa61.js:117
pn.get @ app.4eb2e32d4cd072ffaa61.js:117
pn.run @ app.4eb2e32d4cd072ffaa61.js:117
hn @ app.4eb2e32d4cd072ffaa61.js:117
(anonymous) @ app.4eb2e32d4cd072ffaa61.js:117
Je @ app.4eb2e32d4cd072ffaa61.js:117
Promise.then (async)
Ue @ app.4eb2e32d4cd072ffaa61.js:117
nt @ app.4eb2e32d4cd072ffaa61.js:117
(anonymous) @ app.4eb2e32d4cd072ffaa61.js:117
pn.update @ app.4eb2e32d4cd072ffaa61.js:117
fe.notify @ app.4eb2e32d4cd072ffaa61.js:117
set @ app.4eb2e32d4cd072ffaa61.js:117
vn.set @ app.4eb2e32d4cd072ffaa61.js:117
(anonymous) @ editor-markdown.ebde2da88cff6e2358f4.js:2
y @ vendor.d02ffd96c7099ceb7085.js:2
S @ vendor.d02ffd96c7099ceb7085.js:2
x @ vendor.d02ffd96c7099ceb7085.js:2
setTimeout (async)
(anonymous) @ vendor.d02ffd96c7099ceb7085.js:2
k @ vendor.d02ffd96c7099ceb7085.js:2
ze @ app.4eb2e32d4cd072ffaa61.js:117
n @ app.4eb2e32d4cd072ffaa61.js:117
ze @ app.4eb2e32d4cd072ffaa61.js:117
e.$emit @ app.4eb2e32d4cd072ffaa61.js:117
(anonymous) @ vendor.d02ffd96c7099ceb7085.js:2
(anonymous) @ vendor.d02ffd96c7099ceb7085.js:2
(anonymous) @ vendor.d02ffd96c7099ceb7085.js:2
(anonymous) @ vendor.d02ffd96c7099ceb7085.js:2
Ur @ vendor.d02ffd96c7099ceb7085.js:2
(anonymous) @ vendor.d02ffd96c7099ceb7085.js:2
so @ vendor.d02ffd96c7099ceb7085.js:2
po @ vendor.d02ffd96c7099ceb7085.js:2
replaceRange @ vendor.d02ffd96c7099ceb7085.js:2
insertAtCursor @ editor-markdown.ebde2da88cff6e2358f4.js:2
(anonymous) @ editor-markdown.ebde2da88cff6e2358f4.js:2
ze @ app.4eb2e32d4cd072ffaa61.js:117
e.$emit @ app.4eb2e32d4cd072ffaa61.js:117
insert @ editor.e2e61c72009280dc4d52.js:2
ze @ app.4eb2e32d4cd072ffaa61.js:117
n @ app.4eb2e32d4cd072ffaa61.js:117
ze @ app.4eb2e32d4cd072ffaa61.js:117
e.$emit @ app.4eb2e32d4cd072ffaa61.js:117
click @ app.4eb2e32d4cd072ffaa61.js:193
ze @ app.4eb2e32d4cd072ffaa61.js:117
n @ app.4eb2e32d4cd072ffaa61.js:117
s._wrapper @ app.4eb2e32d4cd072ffaa61.js:117
```

**Comment by onen9ne** [comment]
@NGPixel care to tell me what is wrong?

**Comment by NGPixel** [comment]
I can't replicate your issue so I can't provide you with a fix at the moment. We'll need to investigate some more.

- What browser are you using? Which version?
- What OS is the wiki running on? Which version?
- Which DB are you using? Which version?
